### DISCUSSION TITLE
qrz and hamqth lookup not working

### DISCUSSION THREAD

**Discussion by oe3ide** [magicbug/Cloudlog#994] [Q&A] [answered]
Hi,
I am trying to get the callsign-lookup working:
In application/config/config.php I have:

`$config['app_name'] = "Cloudlog";`
`$config['app_version'] = "1.7";`
`$config['directory'] = "/cloudlog";`
`$config['callbook'] = "qrz"; // Options are hamqth or qrz`
(I tried also "hamqth").

The qrz (and hamqth) user & password is set correctly in the config.php.

If I enter a callsign and leave the field with tab, no lookup occures.
(tried FF & Chrome)
Checked also the lookup via URL:
https://xmldata.qrz.com/xml/current/?username=oe3ide;password=********;callsign=pj4ky
This works as expected.

Any ideas?

73 de Ernst
OE3IDE

**Comment by magicbug** [comment]
I'm not 100% sure why, I'm guessing curl is installed, is there anything in the server logs saying what the issue is

**Comment by oe3ide** [comment]
Hey,
I am using a web-hosting package. I guess curl is not installed....
73

**Comment by oe3ide** [comment]
Ok, it seems to be installed... see screenshot.
![Image 4](https://user-images.githubusercontent.com/55683388/115140857-b5496c00-a039-11eb-8bfd-dc6e3fe2d62f.jpg)

**Comment by oe3ide** [comment]
Tested now "hamqth" .. this is working. But qrz lookup still doesn't return data.

**Comment by oe3ide** [comment]
DEBUG - 2021-04-18 09:43:29 --> cURL Class Initialized
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): Entity: line 118: parser error : Opening and ending tag mismatch: meta line 4 and head /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): </head><body id=ERR_ACCESS_DENIED> /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string():        ^ /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): Entity: line 118: parser error : AttValue: " or ' expected /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): </head><body id=ERR_ACCESS_DENIED> /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string():                 ^ /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): Entity: line 118: parser error : attributes construct error /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): </head><body id=ERR_ACCESS_DENIED> /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string():                 ^ /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): Entity: line 118: parser error : Couldn't find end of Start Tag body line 118 /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): </head><body id=ERR_ACCESS_DENIED> /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string():                 ^ /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): Entity: line 136: parser error : Opening and ending tag mismatch: br line 135 and div /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): </div> /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string():       ^ /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): Entity: line 143: parser error : Opening and ending tag mismatch: hr line 138 and body /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): </body></html> /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string():        ^ /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): Entity: line 143: parser error : Opening and ending tag mismatch: div line 125 and html /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): </body></html> /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string():               ^ /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): Entity: line 144: parser error : Premature end of data in tag hr line 123 /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string():  /home/.sites/805/site1069704/web/cloudlog/application/libraries/Qrz.php 24
ERROR - 2021-04-18 09:43:30 --> Severity: Warning --> simplexml_load_string(): ^

**Comment by oe3ide** [comment, marked as answer]
Hi,
it works now. Solution: I needed a firefall exclusion on my webhoster for qrz and hamqth.
73 Ernst

**Reply by magicbug** [reply]
Perfect

**Reply by pa2eon** [reply]
Dear Ernst,
What are the extra firewall settings at your location .. at mine it takes to long if the QTH information is collect by cloudlog.
Regards, Eric / PA2EON

**Comment by MikleBas** [comment]
> 

Hey!
Where to register login and password from hamqth?
I will try too
Thanks!

**Comment by pa2eon** [comment]
In the config file of the application .. search for behind the application
folder.

Vy73 Eric, PA2EON

Op zo 6 feb. 2022 17:49 schreef MikleBas ***@***.***>:

**Comment by n5tti** [comment]
Mine performs the lookup, but takes almost **2 full minutes** to complete. (with qrz or hamqth)
the firewall has exceptions and i can sucessfully curl xmldata.qrz.com

Thanks
Tom 
N5TTI

**Reply by magicbug** [reply]
If you are having an issue please open a separate thread and include as much information as possible in regard to your setup.
